onedal/cluster/kmeans.py

```diff
@@ -148,6 +148,7 @@ def _get_onedal_params(self, dtype=np.float32):
             "max_iteration_count": self.max_iter,
             "cluster_count": self.n_clusters,
             "accuracy_threshold": thr,
+            "result_options": result_option,
```

Comment:
Or you can update signature and make None as a default param for `result_option` and do check if result_option is None or not. If none use "". As an option. Just avoid using empty strings as default value of params